src/System.Security.Cryptography.Xml/tests/Samples/SigningVerifyingX509Cert.cs

```diff
@@ -60,6 +60,13 @@ private static bool VerifyXml(string signedXmlText, X509Certificate2 certificate
         }
 
         [Fact]
+        [SkipOnTargetFramework(
+            TargetFrameworkMonikers.Net45 |
+            TargetFrameworkMonikers.Net451 |
+            TargetFrameworkMonikers.Net452 |
+            TargetFrameworkMonikers.Net46 |
```

Comment:
I don't think that day would ever come, but this is fine.  (I'm hoping that when we can rebaseline to a net462 or higher we just delete all of the old framework-referencing values, and make the test library nice and clean again)